src/rpc/mempool.cpp

```diff
@@ -669,6 +670,133 @@ static RPCHelpMan savemempool()
     };
 }
 
+static RPCHelpMan submitrawpackage()
+{
+    return RPCHelpMan{"submitrawpackage",
+        "\nSubmit a package of raw transactions (serialized, hex-encoded) to local node (-regtest only). Experimental, API may be unstable.\n"
+        "\nThe package will be validated and accepted to mempool if it passes consensus and mempool policy rules. Throws a JSONRPCError if anything goes wrong.\n"
+        "\nWarning: successful submission does not mean the transaction will propagate to other nodes on the network.\n",
```

Comment:
To inform the package user, you could link back to `doc/policy/packages.md` somewhere.

Reply:
Added